### ISSUE TITLE
database

### ISSUE THREAD

**Issue by sgroy10** [stackblitz/bolt.new#690]
My problem is i love bolt , all my exciting ideas just go to waste because of the database proble, i tried supabase ,firebase , nothing integrates well with all my app, and i hate to leave it mid way. Please integrate inbuilt database or guide a onboarding system in few clicks to connect and take care of errors, my jewelcollab project got stuck with upload function so i had to move out. PLease fix this .

**Comment by mzasmal** [comment]
I agree whole heartedly. I spent 1 fantastic hour building a front end with local storage but when i tried adding supabase or sqllite or just reading and writing data from a google sheet it fell flat. Once this is done it will be a super powerful tool

**Comment by TheWarriorTrader** [comment]
In short, i have the same big problem. Without db all app are only for fun!

**Comment by CHR-JS** [comment]
same for me

**Comment by SiMON-LVLUP** [comment]
I do also have the same problem. I need to make it work !

**Comment by erhanacikgoz11** [comment]
we cannot use database :@
